src/Main.elm

```diff
@@ -561,3 +531,42 @@ renderTime zone time =
 getMillis : Time.Zone -> Time.Posix -> Int
 getMillis zone time =
     Time.toMillis zone time
+
+
+type alias RGBA =
```

Comment:
Note: having elm-ui's `rgba255`/`rgb255` types vs this `RGBA` type alias does get confusing.

We should have a helper functionality so we can at least pass `rgba255` into the `viewIcon` function.

Something like:

```
convertRgba255ToRGBA: Element.rgba255 -> RGBA
... 
```

Reply:
Note: turns out that elm-ui's `toRgb` does just that.

https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/Element#toRgb